packages/snage/src/config/migrate.ts

```diff
@@ -58,6 +58,12 @@ export const findIndex = (contents: YAMLMap, key: string): O.Option<number> => {
     return index === -1 ? O.none : O.some(index);
 };
 
+export const asYAMLMap = (obj: object): YAMLMap =>
+    Object.keys(obj).reduce((map, key) => {
+        map.set(key, obj[key]);
+        return map;
+    }, new YAMLMap());
+
```

Comment:
We could probably also use [`YAML.createNode(obj)`](https://eemeli.org/yaml/#yaml-createnode). But we would need to cast the result from `Node` to `YAMLMap`.

```suggestion
export const asYAMLMap = (obj: Record<string, unknown>): YAMLMap => YAML.createNode(obj) as YAMLMap;
```

Reply:
Well, no need to cast anything, if we directly use `YAML.createNode()` without wrapping it in another method I guess.